webapp/components/logged_in.jsx

```diff
@@ -139,6 +110,45 @@ export default class LoggedIn extends React.Component {
             }
         });
 
+        // Pervent backspace from navigating back a page
+        $(window).on('keydown.preventBackspace', (e) => {
+            if (e.which === BACKSPACE_CHAR && !$(e.target).is('input, textarea')) {
+                e.preventDefault();
+            }
+        });
+    }
+
+    componentWillMount() {
```

Comment:
Can we move all this into the constructor instead of being in `componentWillMount`?